.github/workflows/code_review.yml

```diff
@@ -12,9 +12,10 @@
       - name: Checkout repository
         uses: actions/checkout@v3
       - name: Code Review
-        uses: freeedcom/ai-codereviewer@main
+        uses: ./
         with:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
-          OPENAI_API_MODEL: "gpt-4-1106-preview"
-          exclude: "yarn.lock,dist/**"
+          INFERENCE_API_KEY: ${{ secrets.INFERENCE_API_KEY }}
+          MODEL: "gpt-4o-mini"
+          exclude: "yarn.lock, dist/**"
```

Comment:
The `exclude` pattern has changed slightly with the addition of a space after the comma. Ensure this doesn't affect the file exclusion logic.